lib/encoding.neon

```diff
@@ -0,0 +1,132 @@
+%|
+ |  File: encoding
+ |
+ |  Functions for encoding and decoding ASCII or Binary data in various formats.
+ |%
+IMPORT bitwise
+
+EXPORT base64Encode
+EXPORT base64Decode
+
+TYPE State IS ENUM
+    BYTE1
```

Comment:
Don't use all uppercase for these. Neon style is to use lower case. `byte1`, `byte2`, etc.